tests/test_fetch_data.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+from unittest.mock import patch, MagicMock
+
+# import os
+from dags.scripts.fetch_data import (
+    load_cities_from_csv,
+    fetch_weather_data,
+    fetch_weather_for_cities,
+)
+
+# API_KEY = os.getenv("API_KEY")
+
+
+# Test load_cities_from_csv
+def test_load_cities_from_csv(tmp_path):
+    csv_file = tmp_path / "cities.csv"
+    data = {"city_id": range(1050), "city_name": [f"City{i}" for i in range(1050)]}
+    df = pd.DataFrame(data)
+    df.to_csv(csv_file, index=False)
+
+    result_df = load_cities_from_csv(csv_file)
+    assert len(result_df) == 1000
+    assert list(result_df.columns) == ["city_id", "city_name"]
+
+
+# Test fetch_weather_data
+@patch("dags.scripts.fetch_data.requests.get")
+def test_fetch_weather_data(mock_get):
+    mock_response = MagicMock()
+    mock_response.status_code = 200
+    mock_response.json.return_value = {"list": [{"id": 1, "weather": "sunny"}]}
+    mock_get.return_value = mock_response
+
+    city_group = "1,2,3"
```

Comment:
add a comment above this explaining what `1,2,3` would mean to a new person reading this code